README.md

```diff
@@ -7,21 +7,25 @@ Application in Django with super user admin
 - Kevin Eslava Mantilla - 1152066 - Kevinalexisesma@ufps.edu.co
 - Marlon Diaz Torres - 1152054    - Marlonjosedito@ufps.edu.co
 
-- ## installation steps
+## installation steps
 
-- C:\Users\Asus-PC\Downloads\todoproject>.\venv\Scripts\activate.bat
--(venv) C:\Users\Asus-PC\Downloads\todoproject>pip install django
--(venv) C:\Users\Asus-PC\Downloads\todoproject>py
-    Python 3.12.2 (tags/v3.12.2:6abddd9, Feb 6 2024,21:26:36) [MSC v.1937 64 bit (AMD64)] on win32
-    Type "help", "copyright", "credits" or "license" for more information.
+- .\todoproject>.\venv\Scripts\activate.bat
+- (venv) .\todoproject>pip install django
+- (venv) .\todoproject>py
+
+Inside the python console:
```

Comment:
I don't need you to tell me how to install Django but how to install your app. You should be talking about how to install the dependencies located in the requirements.txt file.